extra/fuegogin/adaptor_test.go

```diff
@@ -0,0 +1,33 @@
+package fuegogin_test
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/go-fuego/fuego/extra/fuegogin/lib"
+)
+
+func TestFuegoGin(t *testing.T) {
+	e, _ := lib.SetupGin()
+
+	t.Run("simply test gin", func(t *testing.T) {
+		r := httptest.NewRequest("GET", "/gin", nil)
+		w := httptest.NewRecorder()
+
+		e.ServeHTTP(w, r)
+
+		require.Equal(t, 200, w.Code)
+	})
+
+	t.Run("test fuego plugin", func(t *testing.T) {
+		r := httptest.NewRequest("GET", "/fuego", nil)
+		w := httptest.NewRecorder()
+
+		e.ServeHTTP(w, r)
+
+		require.Equal(t, 200, w.Code)
+		require.Equal(t, `{"message":"Hello "}`, w.Body.String())
```

Comment:
https://pkg.go.dev/github.com/stretchr/testify/assert#JSONEq

This would provide a better error reporting if the test fails

Reply:
Thanks for the info!